src/style.css

```diff
@@ -1,3 +1,132 @@
 body {
   margin: 0;
+  display: block;
+  padding: 0;
+  box-sizing: border-box;
+
+  overflow-x: hidden;
+}
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  margin: 0 auto;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
```

Comment:
Avoid fixing container sizes like width and height (lines 11 and 12) unless there is a specific requirement for it. This can lead to content overflow or unexpected scrollbars if the content exceeds these dimensions. Consider using min-height or padding instead to ensure the header has enough space for its content.